tests/foreman/longrun/test_oscap.py

```diff
@@ -300,6 +318,134 @@ def test_positive_push_updated_content(self):
                 vm.execute_foreman_scap_client()
                 self.assertTrue(self.oscapreports.search(host))
 
+    @upgrade
+    @run_only_on('sat')
+    @tier4
+    def test_positive_oscap_run_with_tailoring_file_and_capsule(self):
+        """ End-to-End Oscap run with tailoring files and default capsule
+
+        :id: 346946ad-4f62-400e-9390-81817006048c
+
+        :setup: scap content, scap policy, tailoring file, host group
+
+        :steps:
+
+            1. Create a valid scap content
+            2. Upload a valid tailoring file
+            3. Create a scap policy
+            4. Associate scap content with it's tailoring file
+            5. Associate the policy with a hostgroup
+            6. Provision a host using the hostgroup
+            7. Puppet should configure and fetch the scap content
+               and tailoring file
+
+        :CaseAutomation: notautomated
+
+        :expectedresults: ARF report should be sent to satellite reflecting
+                         the changes done via tailoring files
+
+        :CaseImportance: Critical
+        """
+        if settings.rhel7_repo is None:
+            self.skipTest('Missing configuration for rhel7_repo')
```

Comment:
perhaps use decorator @skip_if_not_set('rhel7_repo')

Reply:
I don't think works as , rhel7_repo options doesn't come under any section. The error I get 
`E   ValueError: Feature(s): "rhel7_repo" not found. Available ones are: "ipa, fake_manifest, ec2, compute_resources, vmware, upgrade, rhev, ssh_client, fake_capsules, performance, rhai, discovery, oscap, ldap, capsule, ostree, vlan_networking, transition, clients, server, bugzilla, distro, docker".`